File: API/Helpers/SearchParams.cs
namespace API.Helpers
{
    public class SearchParams : PaginationParams
    {
        public int StoreId { get; set; }
        public string StoreName { get; set; }
        public int ItemId { get; set; }
        public string ItemName { get; set; }
        public int SupplierId { get; set; }
        public string SupplierName { get; set; }

       
        public string OrderBy { get; set; } = "lastActive";
    }
}

File: client/src/app/_models/searchParams.ts


export class SearchParams {
    storeId: number;
    storeName="";
    itemId: number;
    itemName: string;
    itemCategory: string;
    
    supplierId: number;
    supplierName: string;
    supplyTitle:string;
    pageNumber = 1;
    pageSize = 5;
    orderBy = 'lastActive';

    constructor(
        //user: User
        ) {
      //  this.gender = user.gender === 'female' ? 'male' : 'female';
    }
}